tasks/main.yml

```diff
@@ -39,6 +39,6 @@
    when: agent_installed.stat.exists == False and (oneagent_installer_script_url is defined) and (oneagent_installer_script_url is not none)
 
  - name: "Install Dynatrace OneAgent"
-   shell: "sh /tmp/dynatrace-oneagent.sh"
+   shell: "sh /tmp/dynatrace-oneagent.sh APP_LOG_CONTENT_ACCESS={{ dynatrace_app_log_content_access }} INFRA_ONLY={{ dynatrace_infra_only }}"
```

Comment:
I'd actually would be use `environment` directive: https://docs.ansible.com/ansible/2.6/user_guide/playbooks_environment.html

Reply:
Hi @holms I verified with agent installer team that those environment variables will not be read so currently it is the only option to pass them.

Reply:
I've also had problems with "environment" directive :) ended up doing the same